.github/workflows/edited-base.yml

```diff
@@ -0,0 +1,46 @@
+# Instead of adding all the jobs to run on a changed base to "edited.yml", we use this intermediate layer.
+# This will make it, that in the case of editing the title or description, there will only be a single skipped job
+# appearing in the checks list, instead of all the jobs below cluttering the output.
```

Comment:
Should link to https://github.com/orgs/community/discussions/64119 and/or https://github.com/orgs/community/discussions/35058